src/drivers/rpm_capture/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+############################################################################
+#
+#   Copyright (c) 2021 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
#   Copyright (c) 2024 PX4 Development Team. All rights reserved.
```

Reply:
It was in all the new files 😄

Reply:
a5f81a19f3f33c50aea23722798cb1ba76cf8a12